python/lsst/meas/astrom/matchOptimisticB/matchOptimisticBContinued.py

```diff
@@ -367,9 +375,9 @@ def _doMatch(self, refCat, sourceCat, wcs, refFluxField, numUsableSources, minMa
         @param[in] numUsableSources  number of usable sources (sources with known centroid
             that are not near the edge, but may be saturated)
         @param[in] minMatchedPairs  minimum number of matches
-        @param[in] maxMatchDist  maximum on-sky distance between reference objects and sources
-            (an lsst.afw.geom.Angle); if specified then the smaller of config.maxMatchDistArcSec or
-            maxMatchDist is used; if None then config.maxMatchDistArcSec is used
+        @param[in] toleranceStruct lsst.pipe.base.Struct containing
```

Comment:
We discourage that naming convention in LSST code. I stretch it a bit sometimes to include "Ptr" for pointers (in C++ code) but even that has raised complaints so I try to use it judiciously. I strongly recommend avoiding "Struct" in this variable name.